src/docs/product/discover-queries/query-builder.mdx

```diff
@@ -46,6 +46,8 @@ The Query Builder syntax is identical to [Sentry's Search syntax](/product/sentr
 
 Use `OR` and `AND` search conditions between filters. However `OR` cannot be used between aggregate and non-aggregate filters. For more details about these conditions, see [Using `OR` and `AND`](/product/sentry-basics/search/#using-or-and-and).
 
+You can also search multiple values on the same key without an operator. When you don't specify, the search terms automatically use an `OR` operator. For example, "x:[value1, value2]" will find the same results as "x:value1 `OR` x:value2". You can learn more in our content covering [Multiple Values on the Same Key](/product/sentry-basics/search/#multiple-values-on-the-same-key).
```

Comment:
This sounds confusing to me:
- Not sure what `without an operator` means, I'd just remove it
- "When you don't specify" - specify what? My first instinct is to read this as "what happens when you don't specify AND/OR", and what happens is we use AND, not OR, which is the opposite of what is suggested

My suggestion would be something like: "You can also search multiple values for the same key by putting the values in a list, e.g. x:[value1, value2], which will find the same results as `x:value1 OR x:value2`".

Reply:
Oh by without an operator we mean without using AND/OR? Can we call that out explicitly then? Since I don't think "operator" is a standard word for those keywords